manticore/core/state.py

```diff
@@ -40,7 +40,7 @@ class Concretize(StateException):
 
     """
 
-    _ValidPolicies = ["MIN", "MAX", "MINMAX", "ALL", "SAMPLED", "ONE"]
+    _ValidPolicies = ["MIN", "MAX", "MINMAX", "ALL", "SAMPLED", "ONE", "PESI", "OPTI"]
```

Comment:
```suggestion
    _ValidPolicies = ["MIN", "MAX", "MINMAX", "ALL", "SAMPLED", "ONE", "PESSI", "OPTI"]
```
Also, maybe explain what "PESSI" and "OPTI" mean?  They're a little opaque.